### PR TITLE
add_labels_in_df: handle base images too

### PR DESCRIPTION
**Pull request by twaugh**
This is required due to:
https://github.com/projectatomic/osbs-client/pull/581

Signed-off-by: Tim Waugh <twaugh@redhat.com>

### CODE REVIEW

File: atomic_reactor/plugins/pre_add_labels_in_df.py [hunk 1]
```diff
@@ -275,19 +276,22 @@ def run(self):
...
-            self.log.error(message)
-            raise RuntimeError(message)
-        else:
-            base_image_labels = config["Labels"] or {}
-
         dockerfile = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
 
         lines = dockerfile.lines
 
+        if re.match('^koji/image-build(:.*)?$', dockerfile.baseimage):
```
Comment (lcarva) [inline atomic_reactor/plugins/pre_add_labels_in_df.py]: I'd rather this case wasn't called out like this...
Why can't we make inspection of base image optional?
Falling back to an empty dict on `KeyError` seems like a much better solution.

Comment (twaugh) [inline atomic_reactor/plugins/pre_add_labels_in_df.py]: But what if the image inspection fails for some reason? e.g. devicemapper issue or whatever.

Comment (lcarva) [inline atomic_reactor/plugins/pre_add_labels_in_df.py]: Looking at the code, `KeyError` is only raised if `docker.errors.NotFound` is raised.
Any other errors should propagate, causing a build failure.

Comment (twaugh) [inline atomic_reactor/plugins/pre_add_labels_in_df.py]: OK, I misunderstood how it behaved. Feel free to change this...

**Comment by lcarva** [comment]
Updated release notes.

**Pull request merged**